lldb/source/Commands/Options.td

```diff
@@ -691,7 +691,10 @@ let Command = "process launch" in {
   def process_launch_plugin : Option<"plugin", "P">, Arg<"Plugin">,
     Desc<"Name of the process plugin you want to use.">;
   def process_launch_working_dir : Option<"working-dir", "w">, Arg<"DirectoryName">,
-    Desc<"Set the current working directory to <path> when running the inferior.">;
+    Desc<"Set the current working directory to <path> when running the inferior. This setting "
+         "applies only to the current `process launch` invocation. You can use the "
+         "`target.launch-working-dir` setting to set a working-dir that is persistent "
+         "across launches.">;
```

Comment:
"This option" - it's not a setting.

"If `target.launch-working-dir` is set and this option is given, the value of this option will be used instead of the setting."